MAUI/TextInputLayout/Container-Type.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Container Type in .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here all about Container Type support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Container Type in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Containers improve the discoverability of input view by creating a contrast between the input view and assistive elements.
+
+## Filled
+
+The background of the input view will be filled with container color, and its stroke (at the bottom edge) color and thickness will be changed based on its states. It can be enabled by setting the `ContainerType` property to `Filled`.
```

Comment:
The background of the input view will be filled with container color, and its stroke (at the bottom edge) color and thickness will be changed based on its states. It can be enabled by setting the `ContainerType` property to `Filled.`

Reply:
Addressed the changes